README.md

```diff
@@ -203,7 +203,28 @@ result.calc_time  # how long it took zxcvbn to calculate an answer,
                   # in milliseconds.
 ````
 
-The optional `user_inputs` argument is an array of strings that zxcvbn will treat as an extra dictionary. This can be whatever list of strings you like, but is meant for user inputs from other fields of the form, like name and email. That way a password that includes a user's personal information can be heavily penalized. This list is also good for site-specific vocabulary — Acme Brick Co. might want to include ['acme', 'brick', 'acmebrick', etc].
+The optional `options` argument is an object that can contain the following optional properties:
+- `user_inputs` is an array of strings that zxcvbn will treat as an extra dictionary. This can be whatever list of strings you like, but is meant for user inputs from other fields of the form, like name and email. That way a password that includes a user's personal information can be heavily penalized. This list is also good for site-specific vocabulary — Acme Brick Co. might want to include ['acme', 'brick', 'acmebrick', etc].
```

Comment:
should probably be `userInputs` and `feedbackMessages` respectively... this is JavaScript.